feature-libs/my-account/.release-it.json

```
@@ -10,7 +10,7 @@
     "publishPath": "./../../dist/my-account"
   },
   "hooks": {
-    "after:version:bump": "cd ../.. && ng build my-account --prod"
+    "after:version:bump": "cd ../.. && ng build my-account --prod && yarn build:schematics"
```

```suggestion
    "after:version:bump": "cd ../.. && ng build my-account --prod && cd feature-libs/my-account && yarn build:schematics"
```

---

Maybe this version is even better

```suggestion
    "after:version:bump": "cd ../.. && yarn build:myaccount"
```